### PR TITLE
Revert "Summary Chapter in 3.1 Re-Order #488"

### PR DESCRIPTION
**Pull request by victoriadrake**
Reverts OWASP/wstg#489

Mistakenly indented titles.

### CODE REVIEW

**Comment by victoriadrake** [comment]
My apologies for messing up the indentation, I read the document too hastily.

@rcorsi You brought up a good point in your original PR. Do the WSTG and MSTG belong on this page as well?

**Comment by kingthorin** [comment]
@victoriadrake is this a draft on purpose?

**Comment by victoriadrake** [comment]
> @victoriadrake is this a draft on purpose?

Yes, I thought @rcorsi might want to weigh in. Do you feel this should be merged then another PR added later?

**Comment by rcorsi** [comment]
Sorry for the delay.  Enjoying the weekend.  :-)

I like the changes.  Though without the OWASP entry seems like the section is just documenting "Other Pen Testing Methodologies".

We should have OWASP entry as the first entry.  I like it there.

**Comment by rcorsi** [comment]
Should the word "Summary" remain at the start of the section?

And the OWASP section could be described as the following in the initial summary list:

OWASP
  - Web Security Testing Guide (WSTG)
  - Mobile Security Testing Guide (MSTG)

... and later documented something like this:

OWASP

This present document (WSTG) describes the OWASP recommended methodology for web application security testing and for mobile device specific recommendations the Mobile Security Testing Guide (MSTG) is also available.

**Comment by victoriadrake** [comment]
Thank you @rcorsi! Hope you're still enjoying your weekend!

**Comment by rcorsi** [comment]
Yes, it was the best weekend in a long time. :-)  Hope yours was good too.

I did find one tiny typo on this line in the summary.

"Mobile Security Testing Guide (WSTG)"

WSTG should be MSTG.  

Regards,

**Comment by rcorsi** [comment]
Yes, it was the best weekend in a long time. :-)  Hope yours was good too.

I did find one tiny typo on this line in the summary.

"Mobile Security Testing Guide (WSTG)"

WSTG should be MSTG.  

Regards,

**Comment by rcorsi** [comment]
Hi @kingthorin and @victoriadrake  Should I make a commit with all the changes?

File: document/3-The_OWASP_Testing_Framework/1-Penetration_Testing_Methodologies.md [hunk 1]
```diff
@@ -2,13 +2,22 @@
...
-- OWASP Testing Guide
-- Penetration Testing Execution Standard
-- PCI Penetration Testing Guide
-- Penetration Testing Framework
-- Technical Guide to Information Security Testing and Assessment (NIST800-115)
-  - Information Systems Security Assessment Framework (ISSAF)
-  - Open Source Security Testing Methodology Manual (OSSTMM)
+- [OWASP](#owasp)
+  - [Web Security Testing Guide (WSTG)](#web-security-testing-guide-wstg)
+  - [Mobile Security Testing Guide (WSTG)](#mobile-security-testing-guide-wstg)
```
Comment (kingthorin) [inline document/3-The_OWASP_Testing_Framework/1-Penetration_Testing_Methodologies.md:7]:
```suggestion
  - [Mobile Security Testing Guide (MSTG)](#mobile-security-testing-guide-wstg)
```

Comment (rcorsi) [reply document/3-The_OWASP_Testing_Framework/1-Penetration_Testing_Methodologies.md]: Looks good @kingthorin !

**Comment by kingthorin** [comment]
So I don't think you're going to be able to add to this PR branch. If you want to make suggestions then we can accept and incorporate them.
https://help.github.com/en/github/collaborating-with-issues-and-pull-requests/commenting-on-a-pull-request

**Comment by rcorsi** [comment]
Hi @kingthorin I had no plans to make any other changes.  Looks good as you have it.

**Comment by kingthorin** [comment]
Thanks!

**Comment by rcorsi** [comment]
@kingthorin So in the end the original PR was merged/committed?
For the other changes a different PR is needed.  Is that what you meant earlier?

**Comment by rcorsi** [comment]
@kingthorin just asking as I see that you did other commits, but these are not merged in.  Not completely clear the workflow required.

**Comment by kingthorin** [comment]
This is a PR that was reverting the other one which was prematurely merged.

I "think" the issue was that [lines 10 and 11 were indented](https://github.com/OWASP/wstg/pull/489/files#diff-b746e97d87081b133697a2b5a5e8bfa5R9-R11) but shouldn't have been. I believe @victoriadrake opened this PR to "fix the fix" so to speak (or at least that's how I'm treating it).

If there's something else you feel needs to be tweaked here, let us know.

**Comment by rcorsi** [comment]
@kingthorin Yes I agree we wanted to correct the indent problem, which is now fixed.
With @victoriadrake we discussed some additional changes for the OWASP entry so that we can briefly describe it and mention the Mobile guide too.
Should I take all those changes and make a new issue and PR?

**Comment by kingthorin** [comment]
If you don't feel this PR sufficiently covers that then suggest some edits as mentioned earlier.

**Comment by rcorsi** [comment]
@kingthorin, @ThunderSon, @victoriadrake Ok, I see that all the edits are also merge in.  That was my earlier confusion, I didn't know if the really good edits added on top of my PR would be included.
Everything looks really good.  Thank You so much!

**Pull request merged**